### PR TITLE
Adds a GitHub Action that publishes the Scaler docs.

### PR DESCRIPTION
**Pull request by rafa-be**

### CODE REVIEW

File: docs/requirements_docs.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
 sphinx
-sphinx-jsonschema
```
Comment (rafa-be) [inline docs/requirements_docs.txt:2]: I had to remove this package, as it's under GPL 2.0, which is not compatible with Apache 2, even as an imported library.

In addition, it does not seem to be required for generating the documentation.

Comment (JamieSlome) [reply docs/requirements_docs.txt:2]: Cool, if not required, we should remove dependencies 👍

**Comment by JamieSlome** [comment]
@rafa-be - can we look into why the unit tests are taking so long to run? Looks like a runaway issue?

**Comment by rafa-be** [comment]
> @rafa-be - can we look into why the unit tests are taking so long to run? Looks like a runaway issue?

Not sure what happened, I don't see any error.

I restarted it.

**Comment by JamieSlome** [comment]
Thanks @rafa-be - we might want to investigate though as we've had this issue before where the tests just runaway and keep running on 👍 Not major, has occurred twice since we pushed out the repo, maybe 3 times max.

**Review by sharpener6 — APPROVED** [review]

**Review by sharpener6 — APPROVED** [review]

**Pull request merged**